src/main/java/Task.java

```diff
@@ -0,0 +1,60 @@
+public class Task {
+
+    private int id;
+    private String taskInfo;
+    private boolean isDone;
+
+    private static int numberOfTasks = 0;
+
+    public Task(String taskInfo) {
+        setTaskInfo(taskInfo);
+        numberOfTasks++;
+        setId(numberOfTasks);
+        this.isDone = false;
+    }
+
+    public void markAsDone() {
+        if (isDone) {
+            System.out.println("Task was already marked as done!");
+        } else {
+            isDone = true;
+            System.out.println("Good job! I've marked it as done!");
+            System.out.println(this.getStatus() + this.taskInfo);
+        }
+    }
+
+    public void markAsUndone() {
```

Comment:
i like how you have the getStatus function, it is a very efficient way of doing the print